src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java

```diff
@@ -110,6 +112,9 @@
  *
  *      synchronized(this) { } // violation ''synchronized' is not followed by whitespace'
  *      synchronized (this) { } // ok
+ *
+ *      String testOne() { return ("a" + "b");} // OK
```

Comment:
Java does not allow directly nested methods. This will still not compile.

Reply:
You are writing a method inside of another method. The two methods you added need to be declared outside of the last curly brace of the existing method.

Reply:
```
 *      synchronized(this) { } // violation ''synchronized' is not followed by whitespace'
 *      synchronized (this) { } // ok
 *
 *  }
 *  public String testOne() { return ("a" + "b");} // OK
 *  public String testTwo() {
 *  return("a" + "b"); // violation 'return' is not followed by whitespace'
 *  }
```

Reply:
Should I change the example? Please guide me on what I can do!

Reply:
Please do https://github.com/checkstyle/checkstyle/pull/11538#discussion_r851731751, we do not need a separate example for this.

Reply:
Changes Done

Reply:
Resolved.